packages/avatar/src/avatar-group.tsx

```diff
@@ -99,9 +99,10 @@ export const AvatarGroup = forwardRef<AvatarGroupProps, "div">(
         {...rest}
         className={cx("chakra-avatar__group", props.className)}
       >
-        {excess && (
+        {excess > 0 && (
           <chakra.span
             className="chakra-avatar__excess"
+            data-testid="avatar-excess"
```

Comment:
I'd like to avoid hvaing `data-testid` in production code, as its only an [escape hatch](https://testing-library.com/docs/dom-testing-library/api-queries#bytestid). 

Could you try `screen.debug()` and paste the content to the top left of https://testing-playground.com/, then select the excess element on the top right and copy paste the selector from bottom left when the testid is gone? It might fall back to `container.querySelector('.chakra-avatar--excess')` or `screen.getByText('+1')`. I'd be fine with either, although I'd prefer the 2nd.

Reply:
makes total sense! I ended up going for testid because I couldn't find a reliable way of detecting that avatar excess is not on the page.
I thought about using `screen.getByText("+", {exact: false})` but wasn't sure if this was a valid assumption that there will only be a `+` on this component if the excess avatar is there.

I will switch out for the querySelector. I tried using that tool but unfortunately still couldn't find a better query

Reply:
imo the test should break if the `+` changes because its a user-facing UI change that, which is exactly what testing-lib concerns itself with :)